ads/_config.js

```diff
@@ -15,235 +15,313 @@
-};
+export const adConfig = {
+  fakead3p: {
+    renderStartImplemented: true,
+  },
 
-/**
- * List of ad networks that will manually call `window.context.renderStart` to
- * emit render-start event when ad actually starts rendering.
- * Please add yourself here if you'd like to do so (which we encourage).
- * @const {!Array<string>}
- */
-export const waitForRenderStart = [
-  'doubleclick',
-  'fakead3p',
-  'inmobi',
-  'smartadserver',
-];
+  a9: {
+    prefetch: 'https://c.amazon-adsystem.com/aax2/assoc.js',
+  },
+
+  accesstrade: {
+    prefetch: 'https://h.accesstrade.net/js/amp/amp.js',
+  },
+
+  adblade: {
+    prefetch: 'https://web.adblade.com/js/ads/async/show.js',
+    preconnect: [
+      'https://staticd.cdn.adblade.com',
+      'https://static.adblade.com',
+    ],
+  },
+
+  adgeneration: {
+    prefetch: 'https://i.socdm.com/sdk/js/adg-script-loader.js',
+  },
+
+  adsense: {
+    prefetch: 'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js',
+    preconnect: 'https://googleads.g.doubleclick.net',
+    clientIdScope: 'AMP_ECID_GOOGLE',
+  },
+
+  adsnative: {
+    prefetch: 'https://static.adsnative.com/static/js/render.v1.js',
+    preconnect: 'https://api.adsnative.com',
+  },
+
+  adstir: {
+    prefetch: 'https://js.ad-stir.com/js/adstir_async.js',
+    preconnect: 'https://ad.ad-stir.com',
+  },
+
+  adtech: {
+    prefetch: 'https://s.aolcdn.com/os/ads/adsWrapper3.js',
+    preconnect: [
+      'https://mads.at.atwola.com',
+      'https://aka-cdn.adtechus.com',
+    ],
+  },
+
+  aduptech: {
+    prefetch: 'https://s.d.adup-tech.com/jsapi',
+  },
+
+  amoad: {
+    prefetch: [
+      'https://j.amoad.com/js/a.js',
+      'https://j.amoad.com/js/n.js',
+    ],
+    preconnect: [
+      'https://d.amoad.com',
+      'https://i.amoad.com',
+      'https://m.amoad.com',
+      'https://v.amoad.com',
+    ],
+  },
+
+  appnexus: {
+    prefetch: 'https://acdn.adnxs.com/ast/ast.js',
+    preconnect: 'https://ib.adnxs.com',
+  },
+
+  atomx: {
+    prefetch: 'https://s.ato.mx/p.js',
+  },
+
+  caprofitx: {
+    prefetch: [
+      'https://cdn.caprofitx.com/pfx.min.js',
+      'https://cdn.caprofitx.com/tags/amp/profitx_amp.js',
+    ],
+    preconnect: 'https://ad.caprofitx.adtdp.com',
+  },
+
+  colombia: {
+    prefetch: 'https://static.clmbtech.com/ad/commons/js/colombia-amp.js',
+  },
+
+  criteo: {
+    prefetch: 'https://static.criteo.net/js/ld/publishertag.js',
+    preconnect: 'https://cas.criteo.com',
+  },
+
+  dotandads: {
+    prefetch: 'https://amp.ad.dotandad.com/dotandadsAmp.js',
+    preconnect: 'https://bal.ad.dotandad.com',
+  },
+
+  doubleclick: {
+    prefetch: [
+      'https://www.googletagservices.com/tag/js/gpt.js',
+      'https://securepubads.g.doubleclick.net/static/glade.js',
+    ],
+    preconnect: [
+      'https://partner.googleadservices.com',
+      'https://tpc.googlesyndication.com',
+    ],
+    clientIdScope: 'AMP_ECID_GOOGLE',
+    renderStartImplemented: true,
```

Comment:
Yes. We just need to change its default value to `true`.

However, I think that probably will never happen, because it's not free for 3P ad networks.